pipeline/splicing/calcVarPriors.py

```diff
@@ -17,9 +17,95 @@
 import subprocess
 import tempfile
 import os
+import pyhgvs
+import pyhgvs.utils as pyhgvs_utils
+from pygr.seqdb import SequenceFileDB
 from Bio.Seq import Seq
 from calcMaxEntScanMeanStd import fetch_gene_coordinates, runMaxEntScan
 
+'''
+GENERAL NOTES ON REFSEQ NUMBERING AND SPLICING
+
+MINUS VS PLUS STRAND NUMBERING
+Genes are transcribed from 5' to 3'
+For plus strand genes (e.g. BRCA2) this correlates to reading from left to right
+    so genomic position increases from left to right
+This means that for plus strand genes the exon start position is ALWAYS less than
+    the exon end position and that native splice acceptors have a lower genomic position
+    than native splice donors in the same exon
+For minus strand genes (e.g. BRCA1) this correlates to reading from right to left
+    so genomic position increases from right to left AND decreases from left to right
+This means that for minus strand genes the exon start position is ALWAYS greater than
+    the exon end position and that native splice acceptors have a higher genomic position
+    than native splice donors in the same exon
+
+REFSEQ NUMBERING
+RefSeq numbering starts to the right of the first base
```

Comment:
does refseq numbering start to the right of the first base for brca1 and to the left of the first base for brca2? more specifically, in between the first two bases regardless of which direction they're being read?

Reply:
actually, this is still a little confusing based on @melissacline's comment below -- more like it refers to the space just before the first base of the exon and the last base of the intron? specifically, to the right of the last base of the intron and to the left of the first base of the exon for BRCA1 and vice versa for BRCA2?

Reply:
To your first comment, RefSeq numbering starts to the right of the first base for BRCA1 (with genomic coordinates decreasing from left to right) and to the right of the first base for BRCA2 (with genomic coordinates increasing from left to right). Specifically the numbering starts between the first and second base regardless of direction. For your second comment, what you wrote is correct but switch BRCA1 and BRCA2.